packages/serverless-contracts/src/contracts/apiGateway/apiGatewayContract.ts

```diff
@@ -319,4 +321,68 @@ export class ApiGatewayContract<
       params: queryStringParameters,
     });
   }
+
+  getOpenApiDocumentation(): ContractOpenApiDocumentation {
+    const contractDocumentation: OpenAPIV3.OperationObject = {
+      responses: {
+        '200': {
+          description: 'Success',
+          content: {
+            'application/json': {
+              schema: this._outputSchema as OpenAPIV3.SchemaObject,
+            },
+          },
+        },
+      },
+    };
+
+    if (this._pathParametersSchema?.properties !== undefined) {
+      contractDocumentation.parameters = [
+        ...Object.entries(this._pathParametersSchema.properties).map(
+          ([variableName, variableDefinition]) => ({
+            name: variableName,
+            in: 'path',
+            schema: variableDefinition as OpenAPIV3.SchemaObject,
+            required:
+              this._pathParametersSchema?.required?.includes(variableName) ??
+              false,
+          }),
+        ),
+        ...(contractDocumentation.parameters ?? []),
+      ];
+    }
+
+    if (this._queryStringParametersSchema?.properties !== undefined) {
+      contractDocumentation.parameters = [
+        ...Object.entries(this._queryStringParametersSchema.properties).map(
+          ([variableName, variableDefinition]) => ({
+            name: variableName,
+            in: 'query',
+            schema: variableDefinition as OpenAPIV3.SchemaObject,
+            required:
+              this._queryStringParametersSchema?.required?.includes(
+                variableName,
+              ) ?? false,
+          }),
+        ),
+        ...(contractDocumentation.parameters ?? []),
+      ];
+    }
+
+    if (this._bodySchema !== undefined) {
+      contractDocumentation.requestBody = {
+        content: {
+          'application/json': {
+            schema: this._bodySchema as OpenAPIV3.SchemaObject,
+          },
+        },
+      };
+    }
+
+    return {
+      path: this._path,
```

Comment:
Do you need to return the path and method as well? I don't see them used in the plugin

Reply:
J'avais un fail dans la génération de la doc, les paths était mal placé dans le json généré, et pour le faire bien j'avais besoin du path, tu peux voir dans le dernier commit que j'ai ajouté